frontend/src/metabase-lib/parameters/utils/parameter-values.js

```diff
@@ -32,7 +32,12 @@ export function getValuePopulatedParameters({
   parameters,
   values = {},
   defaultRequired = false,
+  collectionPreview = false,
 }) {
+  if (collectionPreview) {
+    return [];
+  }
```

Comment:
It makes me suspicious that we exclude all parameters, but as I am reasoning through:

1. no filters are set — no parameters have values
2. default values should be populated on the backend
3. required parameters/variables can't exist without a default so 2 applies here

In case the logic above is correct, all e2e tests must pass. Otherwise something _might_ fail.

Reply:
do you think we should have some comment explaining this?

Reply:
Yeah I guess it won't do any harm.